test-framework/core/src/main/java/org/keycloak/test/framework/injection/Registry.java

```diff
@@ -288,17 +287,24 @@ private InstanceContext getDeployedInstance(RequestedInstance requestedInstance)
     }
 
     private void loadSuppliers() {
-        Iterator<Supplier> supplierIterator = ServiceLoader.load(Supplier.class).iterator();
+        Iterator<TestFrameworkExtension> extensions = ServiceLoader.load(TestFrameworkExtension.class).iterator();
+        ValueTypeAlias valueTypeAlias = new ValueTypeAlias();
+        List<Supplier> tmp = new LinkedList<>();
```

Comment:
@stianst Don't you want to change it to more nicer name like suppliers?

Reply:
No, because there is a field suppliers, and this is just a temporary list used to build that list.